### PR TITLE
Jak 2: Finnish translations

### LINKED ISSUE #3620: [jak2] [progress] do not use option index as language selection

### PR DESCRIPTION
**Pull request by Aloqas**
Finnish translations for Jak 2. These include cutscenes and all game text.

All subtitle timings for cutscenes as well as non-cutscenes have been edited for a better flow and to fit the 4x3 ratio.
I've been working on these solo for the most part so any input from other finns would be appreciated.

A few issues in the progress menu I mentioned in #3504 still persist

I couldn't figure out how to add Finnish to the options menu, so I'm gonna need someone else to do that part. 💀
But I was able to add them to the debug menu.

I also increased subtitle heap so hopefully that doesn't break anything.

Fixes #3620

### CODE REVIEW

File: decompiler/config/jak2/all-types.gc [hunk 1]
```diff
@@ -8774,6 +8774,7 @@ Unlike [[engine]], users can use [[engine-pers]] as a parent class."
   (progress-graphics-ps2-lod-dist #x1337)
   (progress-graphics-msaa-off #x1338)
   (progress-graphics-display #x1339)
+  (language-name-finnish #x133a)
```
Comment (xTVaser) [inline decompiler/config/jak2/all-types.gc:8777]: you need to also add this to `decompiler/config/jak2/all-types.gc` and the failing builds will pass.

**Comment by 545u** [comment]
> I've been working on these solo for the most part so any input from other finns would be appreciated.

LGTM!

**Comment by xTVaser** [comment]
Conflicts need to be resolved before this can be merged

**Comment by xTVaser** [comment]
Added to menus and tested, id encourage double checking my work though:
image
Screenshot 2024-07-31 223417

**Comment by Aloqas** [comment]
Added the country flag
2024-08-04T16-34-23

**Comment by xTVaser** [comment]
I think you can ignore the failing build, there's a mistake in it that I need to fix.

Is this ready for review now or are you still working on it?

**Comment by Aloqas** [comment]
I think it's ready now

**Comment by xTVaser** [comment]
ok to test

**Review by xTVaser — APPROVED** [review]

**Pull request merged**